scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -450,4 +549,49 @@ mod tests {
             .await
             .unwrap();
     }
+
+    #[tokio::test]
+    #[should_panic]
+    async fn test_checking_timestamps() {
+        let schema = TestTableSchema {
+            name: "COMPARE_TIME".to_string(),
+            partition_key: vec![("pk", "int")],
+            clustering_key: vec![("ck", "int")],
+            other_columns: vec![("v1", "int"), ("v2", "boolean")],
```

Comment:
I was really hoping that the test would also check some more interesting types like tuples, nested collections (after all there was a bug with nested collections in the initial version of this PR), and also non-frozen collections (to verify that `compare_timestamps` doesn't fail if there is a non-frozen collection column but correctly skips it).

Reply:
My bad. I have added a quick test with those.